### DISCUSSION TITLE
Not opening issue, search behavior issue

### DISCUSSION THREAD

**Discussion by arslanarif786** [tailwindlabs/headlessui#3129] [Help]
**What package within Headless UI are you using?**

@headlessui/vue

**What version of that package are you using?**

v1.6.4

**What browser are you using?**

Chrome, Firefox

**Describe your issue**

1. As it is not open first time, if starting value is set as empty.
2. if we talk about search behavior, if double click or then more click event happened upon it, then if we are searching an item from list, and it already exists in the list, but still shown no any value in the list.

**Comment by RobinMalfait** [comment]
Hey!

I'm assuming you are talking about the `Combobox` component. The combobox itself doesn't do any filtering, that behavior and implementation is up to you. If you have a minimal reproduction repo we might be able to help.
